internal/debug.go

```diff
@@ -0,0 +1,42 @@
+/*
+	Debug contains information about debug levels and define
+	debug statements
+*/
+package internal
+
+import (
+	"log"
+	"os"
+)
+
+var (
+	// logFlags    = log.Ldate | log.Ltime | log.Lshortfile
+	logFlags    = log.Ldate | log.Ltime
+	info = log.New(os.Stdout, "INFO: ", logFlags)
+	errorLog    = log.New(os.Stderr, "ERROR: ", logFlags)
+	warning     = log.New(os.Stderr, "WARNING: ", logFlags)
+)
+
```

Comment:
Could you add an interface that defines these functions, something like:
```go
type DebugLogger interface {
	LogInfo(string)
	LogError(error)
	LogWarning(string)
	PrintVersion(string)
}
```
This will help a lot with testing later on.